modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -109,6 +110,19 @@ public void transportClientClusterHealth() {
         }
     }
 
+    @Test
+    public void waitingForShards() throws IOException{
+        try (ElasticsearchContainer container = new ElasticsearchContainer(new ImageFromDockerfile()
```

Comment:
Why image here instead of just mounting files?

Reply:
I have changed it to use withClasspathResourceMapping instead of a custom dockerfile. I think this way makes the test a little bit cleaner. The only thing I am not completely convinced is that I could not zip the preload data.